<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageCard</class>
 <widget class="QMainWindow" name="ManageCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manage Your Cards</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleSelect">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>801</width>
      <height>421</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>419</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="tableInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="typeSelect">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Plain Text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hollow Text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mult Choice</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCard">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editCard">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>530</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="cardIndicator">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>No card has been chosen</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
